html/css/custom.css

```diff
@@ -603,8 +602,6 @@ h6 {
 
 #fh5co-home {
   background-color: transparent;
-  background-size: cover;
-  background-attachment: fixed;
```

Comment:
Does this get rid of the parallax effect on the header image? I will check out the branch and check it out.

Reply:
Yep it does. I think we want to try and keep the effect.

Reply:
This will be trickier than I thought. Let me re-work on that and I will make another PR